fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -296,6 +309,87 @@ public DataApiRequest(
     }
 
     /**
+     * To check whether dateTime column request is first one in the sort list or not.
+     *
+     * @param sortColumns  map of columns and its direction
+     *
+     * @return True if dateTime column is first one in the sort list. False otherwise
+     */
+    protected Boolean isDateTimeFirstSortField(Map<String, SortDirection> sortColumns) {
```

Comment:
When we do this change, we can likely collapse this method body into just a single `return` statement, since I don't think the condition will be that complex.